dataservice/api/biospecimen/schemas.py

```diff
@@ -19,6 +19,15 @@
 SAMPLE_PROCUREMENT_ENUM = {"Autopsy", "Biopsy", "Subtotal Resections",
                            "Gross Total Resections", "Blood Draw",
                            "Other"}
+# Codes from https://github.com/EBISPOT/DUO/blob/master/src/ontology/duo.csv
```

Comment:
This url may be the best source of truth, even though I think it's essentially the same as that csv: http://purl.obolibrary.org/obo/duo.owl
We should probably avoid dynamic generation for now so in the case a change occurs that requires some sort of manual migration.